asistente_ladm_col/asistente_ladm_col_plugin.py

```diff
@@ -578,8 +579,7 @@ def show_about_dialog(self):
         self._about_dialog.exec_()
 
     def installTranslator(self):
-        qgis_locale = QLocale(QSettings().value('locale/userLocale'))
         locale_path = os.path.join(PLUGIN_DIR, 'i18n')
         self.translator = QTranslator()
-        self.translator.load(qgis_locale, 'Asistente-LADM_COL', '_', locale_path)
+        self.translator.load(QGIS_LANG, 'Asistente-LADM_COL', '_', locale_path)
```

Comment:
This doesn't allow Qt to load the translator properly, I switched to Spanish but cannot see the GUI in Spanish. 
![translator_not_installed](https://user-images.githubusercontent.com/652785/40423127-28a18656-5e57-11e8-8763-5b29b1eac5ef.png)